src/components/InputBar/InputBar.jsx

```diff
@@ -23,9 +20,7 @@ const mapDispatchToProps = dispatch => ({
 
 });
 
-const InputBar = connect(
+export default connect(
```

Comment:
I think this is totally fine to do, just wanted to better understand the motivation and thought this was related to a linting error. I think in the past ive gotten some linting errors around this direct approach, but it looks to lint fine so LGTM